src/OpenTracing/StartSpanOptions.php

```diff
@@ -40,29 +40,27 @@ final class StartSpanOptions
      * @throws InvalidReferencesSet when there are inconsistencies about the references
      * @return StartSpanOptions
      */
-    public static function create(array $options)
+    public function __construct(array $options)
```

Comment:
I usually don't like passing an array of options in the constructor, mostly because there are a lot of validations/changes happening in such cases (because of types for example) and hence I prefer to proxy that complexity with a static constructor and keep the construct as dumb as possible.